src/components/detail-page/RecommendedFacility.jsx

```
@@ -1,11 +1,12 @@
 import React, { useEffect, useState, useRef } from "react";
 import { useNavigate } from "react-router-dom";
 import axios from "axios";
+import reviewemptyIcon from "../../assets/common/petgray.svg";
 
 const RecommendedFacility = ({ pensionId }) => {
```

ESLint

[error] 'pensionId' is missing in props validation

(react/prop-types)